src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
+            page.StandardOperators.MinusButton.Click();
+            page.AppName.Click();
+            page.Header.SendKeys("2");
+            page.StandardOperators.NegateButton.Click();
+            page.Header.SendKeys(Keys.Equal);
+            Assert.AreEqual("5", page.GetCalculatorResultText());
+            page.Header.SendKeys(Keys.Escape);
+
+            //Verify Decimal
+            page.StandardOperators.NumberPad.Input(3);
+            page.StandardOperators.NumberPad.DecimalButton.Click();
+            page.AppName.Click();
+            page.Header.SendKeys("2");
+            page.Header.SendKeys(Keys.Equal);
+            Assert.AreEqual("3.2", page.GetCalculatorResultText());
+            page.Header.SendKeys(Keys.Escape);
+
+            //Verify Equal
+            page.HistoryPanel.ClearHistory();
+            page.StandardOperators.EqualButton.Click();
+            page.AppName.Click();
+            page.Header.SendKeys(Keys.Equal);
+            Assert.AreEqual("0", page.GetCalculatorResultText());
+            var historyItems = page.HistoryPanel.GetAllHistoryListViewItems();
+            Assert.IsTrue(historyItems[0].Text.Equals("0= 0", StringComparison.InvariantCultureIgnoreCase));
+            Assert.IsTrue(historyItems[1].Text.Equals("0= 0", StringComparison.InvariantCultureIgnoreCase));
+            page.Header.SendKeys(Keys.Escape);
+
+            //Verify Delete
+            page.StandardOperators.NumberPad.Input(3);
+            page.Header.SendKeys("3");
+            Assert.AreEqual("33", page.GetCalculatorResultText());
+            page.StandardOperators.BackSpaceButton.Click();
+            Assert.AreEqual("3", page.GetCalculatorResultText());
+            page.Header.SendKeys(Keys.Backspace);
+            Assert.AreEqual("0", page.GetCalculatorResultText());
+
+            //Verify Clear Entery
+            page.StandardOperators.NumberPad.Input(3);
+            page.Header.SendKeys(Keys.Add);
+            page.Header.SendKeys("3");
+            page.AppName.Click();
+            page.Header.SendKeys(Keys.Delete);
+            Assert.AreEqual("0", page.GetCalculatorResultText());
+            Assert.AreEqual("3 +", page.GetCalculatorExpressionText());
+
+            page.StandardOperators.NumberPad.Input(9);
+            page.Header.SendKeys(Keys.Subtract);
+            page.Header.SendKeys("6");
+            page.StandardOperators.ClearEntryButton.Click();
+            Assert.AreEqual("0", page.GetCalculatorResultText());
+            Assert.AreEqual("3 + 9 Minus (", page.GetCalculatorExpressionText());
+
+            //Verify Clear
+            page.StandardOperators.ClearButton.Click();
+            string sourceD = WinAppDriver.Instance.CalculatorSession.PageSource;
+            if (sourceD.Contains("CalculatorExpression"))
+            {
+                throw new NotFoundException("This test fails; the Calculator Expression should be cleared");
+            }
+            else
```

Comment:
remove this

Reply:
This actually correctly verifies that Clear worked because it checks the calculator expression.  If there isn't a cleared calculator expression then the test fails